Asterisk.2013/Asterisk.NET/Helper.cs

```diff
@@ -754,6 +754,10 @@ internal static ManagerResponse BuildResponse(Dictionary<string, string> attribu
             int hash, hashEvent;
 
             eventType = attributes["event"].ToLower(CultureInfo);
+            if (eventType == "queuememberpause") //temporary fix for Asterisk 13 pause QueueMemberPause

```

Comment:
Ah, OK. I wasn't aware of the change there. This does sort of harp back to one of my main issues with AsterNET as it stands, in that it's no concept of versioning in AMI. You're left to guess which properties/events etc might apply to the version of Asterisk you're using.